metastore/validator.go

```diff
@@ -98,6 +99,23 @@ func (v tableSchemaValidatorImpl) validateIndividualSchema(table *common.Table,
 			return ErrTimeColumnDoesNotAllowHLLConfig
 		}
 
+		// validate enum case
+		if len(column.EnumCases) > 0 {
+			if column.DisableAutoExpand {
+				return ErrEnumCasesNotSupported
+			}
+			enumCaseSet := make(map[string]struct{})
+			for _, enumCase := range column.EnumCases {
+				if column.CaseInsensitive {
+					enumCase = strings.ToLower(enumCase)
+				}
+				if _, exist := enumCaseSet[enumCase]; exist {
```

Comment:
duplicate maybe fine, we just skip it?

Reply:
no, we should enforce this, or we will do more complicated logic to calculate enum id